test/test_cuda_multigpu.py

```diff
@@ -1303,6 +1303,7 @@ def test_cuda_device_memory_allocated(self):
 
         device_count = torch.cuda.device_count()
         current_alloc = [memory_allocated(idx) for idx in range(device_count)]
+        x = torch.ones(10, device="cuda:0")  # noqa: F841
```

Comment:
```suggestion
        _x = torch.ones(10, device="cuda:0")
```
to make it clear it's intentional